cardano-cli/cardano-cli.cabal

```diff
@@ -103,9 +103,12 @@ library
                      , ouroboros-consensus-shelley
                      , ouroboros-network
                      , process
+                     , QuickCheck
                      , scientific
                      , shelley-spec-ledger
+                     , shelley-spec-ledger-test
```

Comment:
We don't want to depend on testing-related libraries in library code.

Reply:
These dependencies are only due for the `Arbitrary` instance for `Shelley.PaymentCredential StandardShelley`.
..which is basically a `data` wrapper around a `Hash`.

So I perhaps should look harder.

Reply:
To clarify -- in this case I don't care about the credential this `Hash` is supposed to represent -- I only need a hash, to fill in this field, to make a completely fake entry.

Reply:
An example from consensus you can base yourself on:
```haskell
import           Cardano.Crypto.Hash (HashAlgorithm)
import qualified Cardano.Crypto.Hash as Hash

import           Cardano.Ledger.Crypto (ADDRHASH, Crypto)

import qualified Shelley.Spec.Ledger.API as SL
import qualified Shelley.Spec.Ledger.Keys as SL (hashWithSerialiser)

mkDummyHash :: forall h a. HashAlgorithm h => Proxy h -> Int -> Hash.Hash h a
mkDummyHash _ = coerce . SL.hashWithSerialiser @h toCBOR

mkKeyHash :: forall c discriminator. Crypto c => Int -> SL.KeyHash discriminator c
mkKeyHash = SL.KeyHash . mkDummyHash (Proxy @(ADDRHASH c))
```
(The list of imports is probably incomplete)

Reply:
Thank you a lot, Thomas!

Reply:
Fixed, thank you, Thomas!